versioned_docs/version-1.3.0/release-notes/release-notes-for-kubeslice-oss-1.3.1.mdx

```diff
@@ -0,0 +1,17 @@
+# Release Notes for KubeSlice OSS 1.3.1
+
+*Release date: 26th Apr 2024*
+
+KubeSlice is a ***cloud-independent*** platform that combines network, application, Kubernetes, and deployment services 
+in a framework to accelerate application deployment in a multi-cluster and multi-tenant environment. KubeSlice achieves this 
+by creating logical application ***slice*** boundaries that enable seamless communication between pods and services across 
+clusters, clouds, edges, and data centers.
+
+We continue to add new features and enhancements to KubeSlice. 
+
+## What's New :speaker:
+
+Installing KubeSlice in the Air-Gapped Environment using the kubeslice-cli Tool
+
+The kubeslice-cli version 0.6.0 now supports the installation of KubeSlice using local helm chart repositories. Thus supporting the
```

Comment:
Thus supporting the installation in the air-gapped environment.  is not a complete sentence 

Change it to `Thus the tool supports the installation in the air-gapped environment.`

Reply:
done